modules/rostests/apitests/compiler/ms_seh.c

```diff
@@ -89,11 +89,15 @@ START_TEST(ms_seh)
     run_test(seh0006);
     run_test(seh0007);
     run_test(seh0008);
+#ifdef _USE_NATIVE_SEH
```

Comment:
I don't see this define defined anywhere in e.g. some cmake file or a header:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=_USE_NATIVE_SEH